### ISSUE TITLE
sonata_type_model_list for OneToMany or ManyToMany relations

### ISSUE THREAD

**Issue by soullivaneuh** [sonata-project/SonataDoctrineORMAdminBundle#438]
Actually, with a `sonata_type_model_list` type on a ManyToOne relation, we have something like this:

![sonata_many_to_one](https://cloud.githubusercontent.com/assets/1698357/6766385/2d937cac-d005-11e4-8121-1473035c7df5.png)

With a OneToMany or ManyToMany relation, we have a select2 intput like this:

![sonata_many_to_many](https://cloud.githubusercontent.com/assets/1698357/6766390/44512b9c-d005-11e4-844c-a5a0845469ca.png)

It would be very nice to have the same input with the possibility to have an item.

In a nutshell, something like this (sorry, I'm not a photoshop master):

![sonata_combined](https://cloud.githubusercontent.com/assets/1698357/6766408/bf240eac-d005-11e4-9c46-c7a637d916aa.png)

What do you think? Good idea?

Could be possible to implement it easily?

Thanks for reading.

**Comment by ThibautAlgrin** [comment]
I am agree with you. It ils a good idea

**Comment by hason** [comment]
:+1:

**Comment by ThibautAlgrin** [comment]
Hello,
Had you time to finish this feature?

**Comment by Soullivaneuh** [comment]
@ThibautAlgrin Not enough time to look at this now.

Would you like to work on it?

**Comment by ThibautAlgrin** [comment]
@Soullivaneuh OK, I am going to try to work above.
Can you assign me the ticket?

**Comment by Soullivaneuh** [comment]
Only collaborators can be assigned.

No worry, I got note of your message. ;-)

**Comment by wysow** [comment]
:+1: Any update on this?

**Comment by bbelohorec** [comment]
Hi Soullivaneuh, Do you know about solution?

**Comment by Soullivaneuh** [comment]
ping @ThibautAlgrin. :-)

**Comment by ThibautAlgrin** [comment]
Hi, I don't have time to do that. If somebody wants to make it....

**Comment by TaPTaK** [comment]
Somebody already make this?
